nifti2/nifti_tool.h

```diff
@@ -230,83 +254,83 @@ typedef struct {
+NI2_API int    act_run_misc_tests( nt_opts * opts );
+NI2_API int    act_strip      ( nt_opts * opts );  /* strip extras from datasets */
+
+
+NI2_API field_s * get_hdr1_field( const char * fname, int show_fail );
+NI2_API field_s * get_hdr2_field( const char * fname, int show_fail );
+NI2_API field_s * get_nim_field( const char * fname, int show_fail );
+NI2_API const char    * field_type_str (int type);
+
+NI2_API int diff_hdr1s    (nifti_1_header *s0, nifti_1_header *s1, int display);
+NI2_API int diff_hdr1s_list(nifti_1_header *s0, nifti_1_header *s1, str_list *slist,
+                            int display);
+NI2_API int diff_hdr2s    (nifti_2_header *s0, nifti_2_header *s1, int display);
+NI2_API int diff_hdr2s_list(nifti_2_header *s0, nifti_2_header *s1, str_list *slist,
+                            int display);
+NI2_API int diff_nims     (nifti_image *s0,nifti_image *s1,        int display);
+NI2_API int diff_nims_list(nifti_image *s0,nifti_image *s1,str_list *slist,int display);
+
+NI2_API int add_int          (int_list * ilist, int val);
+NI2_API int add_string       (str_list * slist, const char * str);
+NI2_API int check_total_size ( const char *mesg, field_s *fields, int nfields, int tot_size);
+NI2_API int clear_float_zeros( char * str );
+NI2_API int diff_field       (field_s *fieldp, void * str0, void * str1, int nfields);
+NI2_API int disp_cifti_extension ( const char *mesg, nifti1_extension * ext, int maxlen);
+NI2_API int disp_nifti1_extension( const char *mesg, nifti1_extension * ext, int maxlen);
+NI2_API int disp_field       (const char *mesg,field_s *fieldp,void *str,int nfields,int header);
+NI2_API int disp_field_s_list(const char * mesg, field_s *, int nfields);
+NI2_API int disp_nt_opts     ( const char *mesg, nt_opts * opts);
+NI2_API int disp_raw_data    (void * data, int type, int nvals, char space,int newline);
+NI2_API int fill_cmd_string  (nt_opts * opts, int argc, char * argv[]);
+NI2_API int fill_field       (field_s *fp, int type, int offset, int num, const char *name);
+NI2_API int fill_hdr1_field_array(field_s * nh_fields);
+NI2_API int fill_hdr2_field_array(field_s * nh_fields);
+NI2_API int fill_nim1_field_array(field_s * nim_fields);
+NI2_API int fill_nim2_field_array(field_s * nim_fields);
+NI2_API int fill_ana_field_array(field_s * ah_fields);
+NI2_API int modify_all_fields(void *basep, nt_opts *opts, field_s *fields, int flen);
+NI2_API int modify_field     (void * basep, field_s * field, const char * data);
+NI2_API int process_opts     (int argc, char * argv[], nt_opts * opts);
+NI2_API int remove_ext_list  (nifti_image * nim, const char ** elist, int len);
+NI2_API int usage            (char * prog, int level);
+NI2_API int use_full         (void);
+NI2_API int verify_opts      (nt_opts * opts, char * prog);
+NI2_API int write_hdr_to_file (nifti_1_header * nhdr, const char * fname);
+NI2_API int write_hdr2_to_file(nifti_2_header * nhdr, const char * fname);
```

Comment:
These bits might not be needed for nifti_tool, as it isn't a library.

Reply:
That is true probably

Reply:
@afni-rickr So do you mean the whole nifti_tool.h is not needed?

Reply:
@Julianiolo, I just mean that nifti_tool.h is only used in nifti_tool.c, which is a main program file.  It is not used as an external library, so I would not expect there to be a need for any dll or shared library controls.

Reply:
Done :)